neuvector-controller-5.3.advisories.yaml

```diff
@@ -9,31 +9,16 @@ advisories:
       - GHSA-622h-h2p8-743x
     events:
       - timestamp: 2024-04-27T10:34:17Z
-        type: detection
```

Comment:
thanks - though we can keep these existing `detection` events, and append the `false-positive-determination` below it.